os/android/drmhwctwo.h

```diff
@@ -239,6 +239,9 @@ class DrmHwcTwo : public hwc2_device_t {
   hwcomposer::GpuDevice device_;
   std::map<hwc2_display_t, HwcDisplay> displays_;
   std::map<HWC2::Callback, HwcCallback> callbacks_;
+
+ private:
+  static bool is_explicit_sync_enabled_;
```

Comment:
If remove static, there is compile issue which hard to be work around:

vendor/intel/external/android_ia/hwcomposer/os/android/drmhwctwo.cpp:192:29: error: use of non-static data member 'is_explicit_sync_enabled_' of 'DrmHwcTwo' from nested type 'HwcDisplay'
  display_->SetExplicitSync(is_explicit_sync_enabled_);
                            ^~~~~~~~~~~~~~~~~~~~~~~~~
vendor/intel/external/android_ia/hwcomposer/os/android/drmhwctwo.cpp:590:15: error: use of non-static data member 'is_explicit_sync_enabled_' of 'DrmHwcTwo' from nested type 'HwcDisplay'
          if (is_explicit_sync_enabled_) {
              ^~~~~~~~~~~~~~~~~~~~~~~~~
2 errors generated.